Backend/app/rider-platform/rider-app/Main/src/Tools/Notifications.hs

```diff
@@ -572,6 +576,32 @@ notifyOnNewMessage booking message = do
       title = T.pack "Driver"
       body =
         unwords
-          [ message
+          [ decodedMessage
           ]
   notifyPerson person.merchantId notificationData
+
+findLanguage :: Text -> Maybe Maps.Language -> Text
+findLanguage msg language =
+  if isJust $ Map.lookup msg dataMap
+    then do
+      case Map.lookup msg dataMap of
+        Just messages -> do
+          let message = getField (Just Maps.ENGLISH) messages ""
+          if isNothing language
+            then message
+            else getField language messages message
+        Nothing -> msg
+    else msg
+
+getField :: Maybe Maps.Language -> MSG.Messages -> Text -> Text
+getField language messages message = case language of
```

Comment:
far less redundant if we can write like this

`getField :: Maybe Maps.Language -> MSG.Messages -> Text -> Text
getField language MSG.Messages {..} message = case language of

  Just lan ->
    case lan of
      Maps.ENGLISH -> en_us
      Maps.HINDI -> hi_in
      Maps.KANNADA -> kn_in
      Maps.TAMIL -> ta_in
      Maps.MALAYALAM -> ml_in
      Maps.BENGALI -> bn_in
      Maps.FRENCH -> message
  Nothing -> message`

Reply:
is this possible please confirm